config/routes.rb

```diff
@@ -9,6 +9,7 @@
   resources :registrations, only: %i[new create]
 
   resources :articles, only: %i[index show]
+  resource :feedback, only: %i[new create]
```

Comment:
Can we use here?
```suggestion
  resources :feedbacks, only: %i[new create]
```

Reply:
Why resources? We can only create one feedback on this page, and it is not saved anywhere.

Reply:
hmm, ok. we can leave it as is.